src/modules/user/user.service.ts

```diff
@@ -37,14 +43,26 @@ export class UserService {
       length: 10,
       numbers: true,
     });
-    console.log(
-      '-------- The generated password is -------- ',
-      theUser.email,
-      password,
-    );
+
     const salt = await genSalt(10);
     theUser.password = await hash(password, salt);
-    return plainToClass(UserResponseDto, await theUser.save());
+
+    await theUser.save();
+
+    let emailInfo = {
```

Comment:
Just a semantic change, emailInfo is never re-assign, we should use const instead of let